templates/core/build.html

```diff
@@ -5,151 +5,180 @@
-      <dl>
-        <dt>Board</dt>
-        <dd class="js-summary-board"></dd>
-        <dt>Operating system</dt>
-        <dd class="js-summary-os"></dd>
-        <dt>Snaps</dt>
-        <dd><span class="js-summary-snaps">0</span> pre-installed snaps</dd>
-      </dl>
+  <section class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-8">
+        <h2>3 What snaps do you want preinstalled?</h2>
+        <p>Select the applications that will be pre-installed on the device. For Ubuntu Core this will be snaps, for Ubuntu Classic it will be snaps or packages.</p>
+      </div>
     </div>
-    <div class="col-6 p-divider__block">
-      <p>This will trigger a build in Launchpad to create your image. Once the image has been built we will contact us via email on <strong>{{ openid.email }}</strong>. This build will only be available for a few days so please do not rely on Launchpad for future downloads</p>
-      <p>
-        <input class="mktoField" value="yes" id="canonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox">
-        <label class="mktoLabel mktoHasWidth" for="canonicalUpdatesOptIn">I agree to receive information about Canonical’s products and services.</label>
-      </p>
-      <p><button class="p-button--positive js-build-button" aria-disabled="true" disabled="">Build image</button></p>
+    <div class="row p-divider">
+      <div class="col-6 p-divider__block">
+        <form class="p-search-box js-snap-search">
+          <input type="search" class="p-search-box__input" name="search" placeholder="Search for snaps" required="">
+          <button type="reset" class="p-search-box__reset" alt="reset"><i class="p-icon--close"></i></button>
+          <button type="submit" class="p-search-box__button" alt="search"><i class="p-icon--search"></i></button>
+        </form>
+        <div class="js-snap-results"></div>
+      </div>
+      <div class="col-6 p-divider__block">
+        <h4 class="p-muted-heading">Pre-installed snaps</h4>
+        <hr class="u-sv1">
+        <div class="js-preinstalled-snaps-list"></div>
+      </div>
     </div>
-  </div>
-</section>
+  </section>
 
+  <section class="p-strip--light is-shallow">
+    <div class="row">
+      <h2>Summary</h2>
+    </div>
+    <div class="row p-divider">
+      <div class="col-6 p-divider__block">
+        <dl>
+          <dt>Board</dt>
+          <dd class="js-summary-board"></dd>
+          <dt>Operating system</dt>
+          <dd class="js-summary-os"></dd>
+          <dt>Snaps</dt>
+          <dd><span class="js-summary-snaps">0</span> pre-installed snaps</dd>
+        </dl>
+      </div>
+      <div class="col-6 p-divider__block">
+        <p>This will trigger a build in Launchpad to create your image. Once the image has been built we will contact us via email on <strong>{{ openid.email }}</strong>. This build will only be available for a few days so please do not rely on Launchpad for future downloads</p>
```

Comment:
Where did this text come from? “We will contact us”??

Reply:
I wrote it as placeholder text. Fixed the typo at least.